Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.11-slim-buster
+
+LABEL Name='Python Template Python'
+LABEL Version=0.1
```

Comment:
we schould avoid `Version` as it require additional maintenance or workflows to keep it in sync with real version. (anyway build automation will set/override labels)

Reply:
If the LABELs are removed (or set empty), pre-commit throws the following error:
`task: Failed to run task "pre-commit:run": exit status 1`

I am using *.yaml from the existing repo: python-reinforcement-learning (local) where the pre-commit works. I think *.yamls should also be pushed in the template repo. because without them one can spend hours configuring pre-commit.